utils/chain-ops/genesis/testdata/test-deploy-config-full.json

```diff
@@ -60,5 +61,5 @@
   "colosseumMaxTxs": 25,
   "colosseumSegmentsLengths": "3,4",
   "securityCouncilNumConfirmationRequired": 1,
-  "securityCouncilOwners": ["0x007D7e4391DCFdE47Dd7FD0B8E16091C5e0E1C7f","0x2A06Af6a4F325B1e0095a8AbD32888e0AbdCD04D","0xed92E7C40348A552822847384D722A6adB9AFeFA"]
+  "securityCouncilOwners": ["0x007d7e4391dcfde47dd7fd0b8e16091c5e0e1c7f","0x2a06af6a4f325b1e0095a8abd32888e0abdcd04d","0xed92e7c40348a552822847384d722a6adb9afefa"]
```

Comment:
It's because when we use checksummed address, `TestConfigMarshalUnmarshal` fails. But when we use non-checksummded address, the test successes. Do you know why?

Reply:
In the test case `TestConfigMarshalUnmarshal`, it seems like comparing the original JSON string with the JSON representation of the `DeployConfig` object after unmarshaling and marshaling it back to JSON. However, I would suggest considering a more direct approach by comparing the unmarshaled `DeployConfig` object itself with the re-unmarshaled DeployConfig object.

Reply:
TestConfigMarshalUnmarshal reads in hexadecimal format, so uppercase letters are read as lowercase in this process. While it's possible to change the configuration, we'd like to keep the current simple structure for reading JSON as it's a bit more modern, and we don't need a checksum for the test wallet address.

Reply:
Sorry, I stand corrected
JSON does not support HEX, so what I said above is incorrect. The string starting with 0x was losing its uppercase context in the byte array conversion.

Reply:
We've confirmed the above, but it's not a big deal, so here's what we're doing now. @chokobole